homeassistant/components/notify/xmpp.py

```diff
@@ -10,7 +10,10 @@
     ATTR_TITLE, DOMAIN, BaseNotificationService)
 from homeassistant.helpers import validate_config
 
-REQUIREMENTS = ['sleekxmpp==1.3.1', 'dnspython3==1.12.0']
+REQUIREMENTS = ['sleekxmpp==1.3.1',
+                'dnspython3==1.12.0',
+                'pyasn1==0.1.9',
+                'pyasn1-modules==0.0.8']
```

Comment:
sleekxmpp already lists these as requirements in its `setup.py`. Is this just to pin specific version of those libraries?